techniques/general/G223.html

```diff
@@ -47,81 +47,81 @@
-        <ul><li><a href="#technique">About this Technique</a></li>
-<li><a href="#description">Description</a></li>
-<li><a href="#examples">Examples</a></li>
-<li><a href="#related">Related Techniques</a></li>
-<li><a href="#tests">Tests</a></li>
+        <ul><li><a href="#technique">このテクニックについて</a></li>
+<li><a href="#description">解説</a></li>
+<li><a href="#examples">事例</a></li>
+<li><a href="#related">関連テクニック</a></li>
+<li><a href="#tests">検証</a></li>
 </ul>
       </nav>
     </div>
 </aside>
 
 <main id="main" class="standalone-resource__main">
-<h1><span>Technique G223:</span>Using an author-supplied, highly visible focus indicator</h1>
+<h1><span>テクニック G223:</span>コンテンツ制作者が提供した視認性の高いフォーカスインジケータを用いる</h1>
 
 
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
-  <p>This technique is not referenced from any Understanding document.</p>
+  <p>このテクニックはどの解説書からも参照されていない。</p>
 
 
 
-  <p>This technique applies to all technologies.</p>
+  <p>このテクニックは全ての技術に適用される。</p>
 
 </div>
 </section>
 	
 
-<section id="description"><h2>Description</h2>
-    <p>The objective of this technique is enhance the focus indicator in the browser, by creating a highly visible one in the content. This technique is very similar to <a href="G195">G195</a>, but with a stronger indicator to meet <a href="https://w3c.github.io/wcag/understanding/focus-appearance">Focus Appearance</a>.</p>
+<section id="description"><h2>解説</h2>
+    <p>このテクニックの目的は、コンテンツ内に視認性の高いフォーカスインジケータを作成することによって、ブラウザのフォーカスインジケータの視認性を高めることである。このテクニックは <a href="G195">G195</a> と非常に似ているが、<a href="https://waic.jp/translations/WCAG22/Understanding/focus-appearance">フォーカスの外観</a>を満たすために、より目立つインジケータを備えている。</p>
  </section>
  <section id="examples">
-    <h2>Examples</h2>
+    <h2>事例</h2>
     <section class="example" id="example-1-links">
-       <h3>Example 1: Links</h3>
-          <p>A web page has a dark background color and light text and links. When focus lands on a link, the link is outlined with a bright yellow line, 3 pixels wide.</p>
+       <h3>事例 1: リンク</h3>
+          <p>ウェブページの背景色は暗く、テキスト及びリンクは明るい色である。リンクにフォーカスが当たると、リンクは 3 ピクセル幅の明るい黄色の線でアウトライン表示される。</p>
     </section>
     <section class="example" id="example-2-form-elements">
-       <h3>Example 2: Form Elements</h3>
+       <h3>事例 2: フォーム要素</h3>
        
-          <p>A web page includes a form inside a table. The borders of both the table and the form elements are thin, black lines. When focus lands on a form element, the element is outlined with a 5 pixel red line that is partially transparent. The red is equivalent to a hex color of #CA0000, providing a 6:1 contrast ratio with the white background.</p>
+          <p>あるウェブページには、テーブルの中にフォームが含まれている。テーブルとフォーム要素のボーダーはどちらも細い黒線である。フォーカスがフォーム要素に移動すると、その要素は 5 ピクセルの赤い線でアウトライン表示され、部分的に透明になる。この赤は 16 進数で #CA0000 の色に相当し、白い背景とのコントラスト比は 6:1 である。</p>
```

Comment:
こちらも前半の文は G195 そのままで、カラーコードが追加されているだけですね。揃えるならこう:

```suggestion
          <p>テーブルの中にフォームが書かれているウェブページがある。テーブルとフォームの要素の両方の境界は細くて黒い線になっている。フォーカスがフォーム要素に当たる時、その要素は部分的に透明になっている 5 ピクセルの赤い線で囲まれる。この赤は 16 進数で #CA0000 の色に相当し、白い背景とのコントラスト比は 6:1 である。</p>
```